spring-content-s3/src/main/java/internal/org/springframework/content/s3/store/DefaultS3StoreImpl.java

```diff
@@ -55,7 +56,11 @@
 
     private MappingContext mappingContext = new MappingContext(".", ".");
 
-	public DefaultS3StoreImpl(ApplicationContext context, ResourceLoader loader, PlacementService placementService, S3Client client, MultiTenantS3ClientProvider provider) {
+    // reactive
```

Comment:
Any reason for the comments? I think that's pretty clear from the code

Reply:
Not really.  This was all my experimental code.  Removed.